shopfloor_cluster_picking_repack/models/stock_quant_package.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockQuantPackage(models.Model):
+    _inherit = "stock.quant.package"
+
+    def _sync_package_type_from_single_product(self):
```

Comment:
The signature of that one changed:
```suggestion
    def _sync_package_type_from_single_product(self, product, quantity):
```